packages/toolkit/src/login/webview/vue/backend_amazonq.ts

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { SsoConnection } from '../../../auth/connection'
+import { AuthUtil } from '../../../codewhisperer/util/authUtil'
+import { AuthError, CommonAuthWebview } from './backend'
+import { awsIdSignIn } from '../../../codewhisperer/util/showSsoPrompt'
+import { connectToEnterpriseSso } from '../../../codewhisperer/util/getStartUrl'
+
+export class AmazonQLoginWebview extends CommonAuthWebview {
+    // TODO: update source, id
+    public override id: string = 'aws.AmazonCommonAuth'
+    public override source: string = 'src/login/webview/vue/index.js'
```

Comment:
do the child classes need to define the source? can we just pull from the abstract class?